sdk/typescript/src/types/objects.ts

```diff
@@ -423,9 +424,12 @@ export const CheckpointedObjectId = object({
 });
 export type CheckpointedObjectId = Infer<typeof CheckpointedObjectId>;
 
+export const PaginatedNextCursor = object({ objectId: ObjectId });
```

Comment:
I probably wouldn't export this because the next cursor varies quite a bit and this naming might be misleading.

Reply:
Yep, removed the export, I assume I do not need the `type` variable here either since its contained inside here anyway